rusk/src/lib/lib.rs

```diff
@@ -396,7 +402,7 @@ impl Rusk {
 
     fn query<A, R>(
         &self,
-        module_id: ModuleId,
+        module_id: ContractId,
```

Comment:
Should we call the `module_id` herr `contract_id` too or is there a reason to keep the module nomenclature in this context?

Reply:
This is clearly a stray. Will change.